seqseek/chromosome.py

```diff
@@ -44,7 +50,10 @@ def validate_assembly(self):
 
     def validate_name(self):
         if self.name not in self.ASSEMBLY_CHROMOSOMES[self.assembly]:
```

Comment:
Would be nice if these if-statements paralleled the order of preference earlier. First check accession_lengths, then assembly_chromosomes. Logically the same but a bit confusing to read them in opposite orders.